extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubePlaylistExtractor.java

```diff
@@ -327,17 +326,22 @@ public InfoItemsPage<StreamInfoItem> getInitialPage() throws IOException, Extrac
                 .map(content -> content.getObject("itemSectionRenderer")
                         .getArray("contents")
                         .getObject(0))
-                .filter(contentItemSectionRendererContents ->
-                        contentItemSectionRendererContents.has(PLAYLIST_VIDEO_LIST_RENDERER)
-                                || contentItemSectionRendererContents.has(
-                                "playlistSegmentRenderer"))
```

Comment:
Nice catch that "playlistSegmentRenderer" wasn't really used even before!

Reply:
It seems it was used for seasonal playlists: see https://github.com/TeamNewPipe/NewPipeExtractor/commit/8b3f90eb7e4a56ee7e33128af7cf8dd94663414a.

I don't remember now why I have added this though.